pkg/kubelet/network/hairpin/hairpin.go

```diff
@@ -37,7 +37,7 @@ const (
 )
 
 var (
-	ethtoolOutputRegex = regexp.MustCompile("peer_ifindex: (\\d+)")
+	ipLinkOutputRegex = regexp.MustCompile("eth0@if(\\d+)")
```

Comment:
kubenet is not depending on the kubelet hairpin logic now. The CNI bridge plugin does the job.